app/assets/stylesheets/blacklight.scss

```diff
@@ -66,3 +66,7 @@ img.no-icon {
 .page-item span.page-link {
   color: $dark !important;
 }
+
+.blacklight-date_issued_ssi.col-sm-3 {
+  white-space: nowrap;
```

Comment:
"Publication date" was too long to display on a single line on the search page with the default css, so override to keep it inline.